.github/workflows/black.yml

```
@@ -6,17 +6,32 @@ jobs:
   lint:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
 
       - name: Set up Python 3.9
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
```

I think we should use Python `3.11` instead of `3.9`

```suggestion
        uses: actions/setup-python@v5
```